packages/services/api/src/modules/audit-logs/resolvers/Mutation/exportAuditLogsToFile.ts

```diff
@@ -0,0 +1,15 @@
+import type { MutationResolvers } from './../../../../__generated__/types';
```

Comment:
These resolvers should be generated using server preset, so this line should use `types.next.ts` like this:

```
import type { MutationResolvers } from './../../../../__generated__/types.next';
```

I'm not sure what happened, but it seems like server preset is not checking where this type is coming from 🤔  (something for me to check)

Reply:
Ah, if you remove this line and run `codegen` again, server preset should put back the import line with `__generated__/types.next`

Reply:
In this commit, https://github.com/kamilkisiela/graphql-hive/pull/5530/commits/1def8b0f414570e542e83805d7adcd225b7f8532 we are using `types.next` for the rest of the imports